lib/media/streaming_engine.js

```diff
@@ -353,6 +394,19 @@ shaka.media.StreamingEngine = class {
     }
   }
 
+  /**
+   * @property {shaka.util.ManifestParserUtils.ContentType} contentType
+   * Unloads the media state in the event of a MediaSource reload.
```

Comment:
Nit: This line should be indented more. An additional two spaces. See the JSDoc for `switchVariant` for reference.